src/mesh/generated/meshtastic/telemetry.pb.h

```diff
@@ -71,7 +71,9 @@ typedef enum _meshtastic_TelemetrySensorType {
     /* MAX17048 1S lipo battery sensor (voltage, state of charge, time to go) */
     meshtastic_TelemetrySensorType_MAX17048 = 28,
     /* Custom I2C sensor implementation based on https://github.com/meshtastic/i2c-sensor */
-    meshtastic_TelemetrySensorType_CUSTOM_SENSOR = 29
+    meshtastic_TelemetrySensorType_CUSTOM_SENSOR = 29,
+    /* SCD40/SCD41 CO2 sensor */
+    meshtastic_TelemetrySensorType_SCD4X = 30
```

Comment:
These files are generated and will be overwritten with the newxt commit. You need to add this to the protobuf repository instead.

Reply:
That would make sense - created pull request https://github.com/meshtastic/protobufs/pull/572